superset-frontend/cypress-base/cypress/integration/dashboard/load.test.ts

```diff
@@ -23,12 +23,18 @@ import {
 } from './dashboard.helper';
 
 describe('Dashboard load', () => {
-  before(() => {
+  beforeEach(() => {
     cy.login();
-    cy.visit(WORLD_HEALTH_DASHBOARD);
   });
 
   it('should load dashboard', () => {
+    cy.visit(WORLD_HEALTH_DASHBOARD);
     WORLD_HEALTH_CHARTS.forEach(waitForChartLoad);
   });
+
+  it('should load in edit/standalone mode', () => {
+    cy.visit(`${WORLD_HEALTH_DASHBOARD}?edit=true&standalone=true`);
```

Comment:
Would there be any value in testing the other permutations of these params?

Reply:
Sure, I was basically just going for parity with the existing tests that I deleted but I don't see why we can't add more.